lisa/tools/lspci.py

```diff
@@ -184,6 +209,19 @@ def get_gpu_devices(self, force_run: bool = False) -> List[PciDevice]:
         ]
         return gpu_device_list
 
+    def get_devices_by_vendor_device_id(
+        self,
+        vendor_id: str,
+        device_id: str,
+        force_run: bool = False,
+    ) -> List[PciDevice]:
+        full_list = self.get_devices(force_run)
```

Comment:
When call methods, use named arguments, so it's compatible with future changes.

Reply:
Done